source/drafter/build/cargo.d

```diff
@@ -0,0 +1,62 @@
+/* SPDX-License-Identifier: Zlib */
+
+/**
+ * Drafter - Python integration
+ *
+ * Authors: Â© 2020-2022 Serpent OS Developers
```

Comment:
This looks weird -- as if there's an encoding issue?

Feel free to update Copyright year range as well.

Reply:
Fixed :+1: